es6/lib/utils.js

```diff
@@ -272,7 +272,7 @@ var range = function(max) {
 };
 
 var hasKey = function(obj, key) {
-  return typeof obj[key] !== undefined;
+  return typeof obj[key] !== 'undefined';
```

Comment:
Wondering if having undefined as an override is a desired behaviour. If so, then using something like `hasOwnProperty` might also be a good idea.